src/osd/PGLog.h

```diff
@@ -974,7 +1042,12 @@ struct PGLog : DoutPrefixProvider {
       rollbacker,
       this);
   }
+
+  void merge_log_dups(pg_log_t& olog,
+		      bool& changed);
```

Comment:
minor: olog could be const, and convention is to pass out parameters as pointers to make them explicit - since there's just one you could make changed the return value instead